MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -772,6 +781,13 @@ public override string GenerateParameters()
                     parameters.Add($"{FlagStrings.Skip}={SkipValue}");
             }
 
+            // Dump Write Offset
+            if (this[FlagStrings.DumpWriteOffset] == true)
+            {
+                if (DumpWriteOffsetValue != null && DumpWriteOffsetValue > 0)
```

Comment:
Does the write offset have to be positive?

Reply:
No i'm just a dumb copypaste machine